concourse/scripts/kick-off-pipeline.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+#
+# Required variables are:
+# - DEPLOY_ENV
+# - AWS_ACCOUNT
+#
+# Optional variables:
+# - PIPELINES_TO_UPDATE
+
+set -u
+set -e
+
+if [ ! -d "./paas-cf" ]; then
+  echo "Resource paas-cf must be checkout"
+  exit 1
+fi
+
+if [ "${ENABLE_MORNING_DEPLOYMENT}" != "true" ]; then
+  echo "Pipeline kick-off is disabled. Skipping. (set ENABLE_MORNING_DEPLOYMENT=true to enable)"
+else
+  echo "Pipeline kick-off is enabled. Updating. (set ENABLE_MORNING_DEPLOYMENT=false to disable)"
+
+  make -C ./paas-cf "${AWS_ACCOUNT}" run_job JOB="pipeline-lock"
```

Comment:
I don't think this is the right thing to use. The `run_job` make task does 2 things. It unpins the `paas-cf` dependency, and then triggers the job. We don't want this to unpin anything.